doc/api/training/smp_versions/v1.1.0/smd_model_parallel_tensorflow.rst

```diff
@@ -81,7 +81,7 @@ TensorFlow API
       [...]
       x = tf.constant(1.2)                     # placed in partition 0
       with smp.partition(1):
-          y = tf.add(x, tf.constant(2.3))      # placed in partition 1
+          y = tf.add(x, tf.constant(2.3.1))      # placed in partition 1
```

Comment:
I don't think we want to change it here.

Reply:
I removed this update.